### ISSUE TITLE
If a pinned toot has a content warning, its impossible to read it with a screen reader.

### ISSUE THREAD

**Issue by TheQuinbox** [nolanlawson/pinafore#2294]
STR:
1. Open someone's profile with a pinned toot with a CW. In this case, I used mine (thequinbox@dragonscave.space).
2. Press X to try and reveal what's behind the content warning.
3. Attempt to read the toot using a screen reader.
The text of the post is totally ignored.

**Comment by nolanlawson** [comment]
I looked into it, and there doesn't seem to be anything different about pinned toots compared to unpinned toots. When I press X on a pinned toot, it (visually) opens as it should. It also updates the `aria-label` on the `<article>` appropriately.

I think this is a duplicate of https://github.com/nolanlawson/pinafore/issues/2180. I.e. it's a screenreader issue. Closing for now, please reopen if I'm wrong!

**Comment by TheQuinbox** [comment]
Hi,
This does appear to be different, yeah. It doesn't read, even after scrolling over it with J and K. Would an audio demo perhaps help to demonstrate what I mean?

**Comment by nolanlawson** [comment]
@TheQuinbox What screenreader, browser, and OS are you using? I inspected this in the DevTools, and the `aria-label` of the `<article>` is correctly being updated.

**Comment by TheQuinbox** [comment]
Yup, that's the same issue I'm having. Although, with a content warning, the warning is read. However, when you press x, the exact same thing @marcozehe described happens. 

I've tried this with VoiceOver in both Safari and Firefox, and NVDA in Firefox and Edge.

**Comment by nolanlawson** [comment]
Ah okay, somehow pinned toots are missing the aria-label I guess. That seems to be the bug.
